docs/source/package_reference/accelerator.md

```diff
@@ -137,6 +137,34 @@ Use [`~Accelerator.save`] instead of `torch.save`:
 + accelerator.save(state_dict, "my_state.pkl")
 ```
 
+#### `transformers` models
+
+If you are using models from the [🤗 Transformers](https://huggingface.co/docs/transformers/) library, you can use the `.save_pretrained()` method.
+
+```python
+from transformers import AutoModel
+
+model = AutoModel.from_pretrained("bert-base-cased")
+model = accelerator.prepare(model)
+
+# ...fine-tune with PyTorch...
+
+model = accelerator.unwrap_model(model)
+model.save_pretrained(
```

Comment:
```suggestion
unwrapped_model = accelerator.unwrap_model(model)
unwrapped_model.save_pretrained(
```

Reply:
Sure thing. That's actually how my own code looks 😄